src/sentry/static/sentry/app/components/charts/components/legend.tsx

```diff
@@ -23,7 +23,7 @@ export default function Legend(
     padding: 0,
     formatter,
     icon: 'circle',
-    itemHeight: 8,
+    itemHeight: 14,
```

Comment:
This will apply to more than dashboards. Is that ok?

Reply:
`itemHeight: 14` and `lineHeight: 14` were the minimum values I found that worked with the text not being cut off.

This is what `itemHeight: 13` and `lineHeight: 13` looks like with the top parts of the curly parenthesis partially cut off (click on screenshot to see a bigger version):
<img width="1866" alt="Screen Shot 2021-03-08 at 3 35 32 PM" src="https://user-images.githubusercontent.com/139499/110378908-2dc51200-8024-11eb-8ba5-3633b91c8f55.png">